scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultPacBio.java

```diff
@@ -300,17 +311,23 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
       builder.addParameter("instrument", dto.getSequencerName());
       builder.addParameter("run", dto.getRunAlias());
       builder.addParameter(
-          "from", dto.getStartDate().truncatedTo(ChronoUnit.DAYS).format(URL_DATE_FORMAT));
+          "from",
+          dto.getStartDate()
+              .truncatedTo(ChronoUnit.DAYS)
```

Comment:
I think the order needs to swap: put it in the right zone, then truncate.

Reply:
Done